TheMET/TheMET/Network/MetAPI.swift

```diff
@@ -62,42 +63,43 @@ class MetAPI{
             parameters: parameters) { result in
                 switch result {
                 case .failure:
-                    completion(nil)
+                    completion(.failure(.invalidUrlComponents))
```

Comment:
This is not a `invalidUrlComponents` error. This is some error provided by the completion of the `networkManager.get` method. Pls introduce next case to `MetAPIError` and use it:
```
case .networkError(NetworkManagerError)
```